kong/tracing/tracing_context.lua

```diff
@@ -0,0 +1,109 @@
+local ngx = ngx
+
+
+local function init_tracing_context(ctx)
+  ctx.TRACING_CONTEXT = {
+    -- trace ID information which includes its raw value (binary) and all the
+    -- available formats set during headers propagation
+    trace_id = {
+      raw = nil,
+      formatted = {},
+    },
+    -- Unlinked spans are spans that were created (to generate their ID)
+    -- but not added to `KONG_SPANS` (because their execution details were not
+    -- yet available).
+    unlinked_spans = {}
```

Comment:
it is better to use `kong.table.new` to create a new table and give enough slots to avoid re-hash table

Reply:
very good point. Thanks